src/ruby/qps/proxy-worker.rb

```diff
@@ -155,6 +157,10 @@ def proxymain
     opts.on("-c", "--[no-]use_protobuf_c_extension", "Use protobuf C-extention") do |c|
       options[:c_ext] = c
```

Comment:
small nit that is outside of the scope of this PR: we might want to change  `c_ext` key from symbol to string to be consistent with the other keys to the `options` hash.